apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -172,3 +189,47 @@ function ProgressBar(partial: number | null, total: number | null) {
     </div>
   `;
 }
+
+function GraderAssignmentModal({
+  csrfToken,
+  courseStaff,
+}: {
+  csrfToken: string;
+  courseStaff: User[];
+}) {
+  return Modal({
+    id: 'grader-assignment-modal',
+    title: 'Assign instances to graders',
+    body: html`
+      <p>Assign instances to the following graders:</p>
+      ${courseStaff.map(
+        (staff) => html`
+          <div class="form-check">
+            <input
+              type="checkbox"
+              id="grader-assignment-${staff.user_id}"
+              name="assigned_grader"
+              value="${staff.user_id}"
+              class="form-check-input"
+            />
+            <label class="form-check-label" for="grader-assignment-${staff.user_id}">
+              ${staff.name ? `${staff.name} (${staff.uid})` : staff.uid}
+            </label>
+          </div>
+        `,
+      )}
+      <div class="mt-2 small alert alert-info">
+        Only instances that require grading and are not yet assigned to a grader will be affected.
+        If more than one grader is selected, the instances will be randomly split between the
+        graders.
+      </div>
+    `,
+    footer: html`
+      <input type="hidden" name="unsafe_assessment_question_id" value="" />
+      <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+      <input type="hidden" name="__action" value="assign_graders" />
+      <button type="submit" class="btn btn-primary">Assign</button>
+      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
```

Comment:
Was this copied from somewhere? Typically we'd want the cancel/dismiss button on the left and the affirmative action button on the right.

Reply:
I think this was copilot 😬 ... I'll change it.

Reply:
I just double checked, and some other modals in manual grading also use this order: [rubric settings](https://github.com/PrairieLearn/PrairieLearn/blob/781e26415ef7b2a5ff1ebfde3f78ce7413642247/apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts#L260) and [grading conflict info on assessment question](https://github.com/PrairieLearn/PrairieLearn/blob/781e26415ef7b2a5ff1ebfde3f78ce7413642247/apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.ts#L132). Should I change those too?

Reply:
Yes, though maybe in another PR that we can land independently?

Reply:
Moved to #10414.